.eslintrc.json

```diff
@@ -16,7 +16,8 @@
     "prettier/prettier": [
       "error",
       {
-        "singleQuote": true
+        "singleQuote": true,
+        "endOfLine":"auto"
```

Comment:
🎉

Reply:
Do we want `"trailingComma": "all"` here?

Reply:
We can debate this to death, but I always prefer it, makes diffs super clean